src/quickmozview.cpp

```diff
@@ -145,7 +145,13 @@ void QuickMozView::itemChange(ItemChange change, const ItemChangeData &data)
         if (data.window) {
             connect(data.window, &QQuickWindow::contentOrientationChanged, this, &QuickMozView::updateOrientation);
 
-            updateOrientation(data.window->contentOrientation());
+            // Update the orientation, but without emitting an orientationChanged signal
```

Comment:
```
blockSignals(true);
updateOrientation(data.window->contentOrientation());
blockSignals(false);
```

=> Above should do the same. See [1]

https://doc.qt.io/qt-5/qobject.html#blockSignals

Reply:
Thanks: code updated.